include/wx/msw/wrl/EventToken.h

```diff
@@ -0,0 +1,18 @@
+///////////////////////////////////////////////////////////////////////////////
+// Name:        wx/msw/wrl/EventToken.h
+// Purpose:     WRL EventRegistrationToken implementation
+// Author:      Maarten Bent
+// Created:     2021-02-26
+// Copyright:   (c) 2021 wxWidgets team
+// Licence:     wxWindows licence
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef __eventtoken_h__
```

Comment:
Should we use `_WX_MSW_WRL_EVENTTOKEN_H_` here or it important to define this header guard?

Reply:
I used the same guard name as in the Microsoft wrl header, so there will be no type redefinition if for some reason both headers are included.

Reply:
Yes, this is what I suspected, but wasn't sure about it. Maybe it could be worth adding a brief comment explaining it here just to prevent somebody (yes, I often speak of myself in third person, why?) from normalizing it later without thinking?

Reply:
Sure, I'll add a comment